db/migrate/20160817180458_add_move_turn_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddMoveTurnToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :move_turn, :string
```

Comment:
I'd suggest changing the name to something even more explicit.

Maybe something like `current_move_color`. Which I think makes it super explicit what the column is for. :smile: